roles/edpm_nova/tasks/configure.yml

```diff
@@ -12,65 +13,65 @@
     group: "{{ item.group | default(ansible_user) }}"
     mode: "{{ item.mode | default(omit) }}"
   with_items:
-  - { "path": "{{edpm_nova_config_dest}}" }
-  - { "path": "/var/lib/openstack/config/containers"}
-- name: create persistent directories
+    - { "path": "{{ edpm_nova_config_dest }}" }
+    - { "path": "/var/lib/openstack/config/containers" }
+- name: Create persistent directories
   tags:
     - configure
     - nova
   become: true
-  file:
+  ansible.builtin.file:
     path: "{{ item.path }}"
     setype: "container_file_t"
     state: "directory"
     owner: "{{ item.owner | default(omit) }}"
     group: "{{ item.group | default(omit) }}"
     mode: "{{ item.mode | default(omit) }}"
   with_items:
-  - { "path": "/var/lib/nova"}
-  - { "path": "/var/lib/_nova_secontext"}
-  - { "path": "/var/lib/nova/instances"}
-  - { "path": "/var/log/containers/nova", "mode": "0750" }
-  - { "path": "/var/log/containers/stdouts" }
-  - { "path": "/etc/ceph", "mode": "0750", "owner": "root", "group": "root"}
-  - { "path": "/etc/multipath" }
-  - { "path": "/etc/iscsi" }
-  - { "path": "/var/lib/iscsi" }
-  - { "path": "/run/openvswitch" }
-- name: render nova config files
+    - { "path": "/var/lib/nova" }
+    - { "path": "/var/lib/_nova_secontext" }
+    - { "path": "/var/lib/nova/instances" }
+    - { "path": "/var/log/containers/nova", "mode": "0750" }
+    - { "path": "/var/log/containers/stdouts" }
+    - { "path": "/etc/ceph", "mode": "0750", "owner": "root", "group": "root" }
+    - { "path": "/etc/multipath" }
+    - { "path": "/etc/iscsi" }
+    - { "path": "/var/lib/iscsi" }
+    - { "path": "/run/openvswitch" }
```

Comment:
@SeanMooney @gibizer can we define "mode" for these dirs? For iscsi related dirs i can take the values from `edpm_iscsi` role but I'll need your help for other directories. :)

Reply:
so we are not defiening mode so that we don't change it if its already crated by another role.

The nova role needs these directories to exist because we bind mount them in to the nova container.  the other way to do this would be to not define the 
```
   - { "path": "/var/log/containers/stdouts" }
  - { "path": "/etc/ceph", "mode": "0750", "owner": "root", "group": "root"}
    - { "path": "/etc/ceph", "mode": "0750", "owner": "root", "group": "root" }
  - { "path": "/etc/multipath" }
    - { "path": "/etc/multipath" }
  - { "path": "/etc/iscsi" }
    - { "path": "/etc/iscsi" }
  - { "path": "/var/lib/iscsi" }
    - { "path": "/var/lib/iscsi" }
  - { "path": "/run/openvswitch" }
    - { "path": "/run/openvswitch" }
 ```
and assume they are created by other roles

but update the modeclue prepare playbook to create them.

perhaps that is the better approach

by default for directories our modes should be 0755 if they may be share between different services or where traversiablity to sub dirs is a factor.

Reply:
yeah, even "0755" is fine if needed. 

I agree with the path that you are suggesting and i'll fix the molecule prepare playbook as well.

Reply:
just lookign at them /etc/ceph is nolonger used by the container we now use a spereate directory in /var/lib/openstack/config/ceph i belive so you can entirely remove the reference to /etc/cpeh i belive but i would check with @fultonj to confirm

Reply:
I believe Sean is right. If you include that change here the CI should catch it if we're mistaken. It could also be addressed in a follow up patch if we want to limit the scope of this change to satisfying the linter.

Reply:
I'll leave ceph directory as special case since you're right that we can't test it by now  and I'm ensuring that other folders are created by other roles before removing them from here.

Reply:
I added "mode: 0755" only for nova folders, left the other dirs untouched. It shouldn't be an issue since they are created by their own roles.

I'll propose a followup patch in the future.